src/pages/NewChairConsole.js

```diff
@@ -112,14 +117,16 @@ const Records = (props) => {
           return h(TableTextButton, {
             key: `reopen-button-${e.referenceId}`,
             onClick: () => openConfirmation(dar, index),
-            label: 'Re-Open'
+            label: 'Re-Open',
+            disabled: !isChair
           });
```

Comment:
Same idea here, should we even render it if the user can't perform the action?

Reply:
I was trying to follow the logic of the vote buttons since these buttons look the same, and the vote button just get disabled when there is no vote for the user, so that's why I did this but what do you think?

Reply:
I get that. My opinion is that if something isn't useful to the user, then it shouldn't be there. Having a disabled button that will always be disabled (because they're not a chair in the dac) is not useful. It's just noise on the page.

Reply:
makes sense to me!